packages/react-generator/src/templates/react-template/template-files/src/lowcodeConfig/lowcode.js

```
@@ -0,0 +1,25 @@
+ * Copyright (c) 2023 - present TinyEngine Authors.
+ * Copyright (c) 2023 - present Huawei Cloud Computing Technologies Co., Ltd.
+ *
+ * Use of this source code is governed by an MIT-style license.
+ *
+ * THE OPEN SOURCE SOFTWARE IN THIS PRODUCT IS DISTRIBUTED IN THE HOPE THAT IT WILL BE USEFUL,
+ * BUT WITHOUT ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS FOR
+ * A PARTICULAR PURPOSE. SEE THE APPLICABLE LICENSES FOR MORE DETAILS.
+ *
+ */
+
+import React, { createContext, useContext, useState } from 'react';
```

ESLint

[error] 'useState' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)